bskyweb/cmd/embedr/server.go

```diff
@@ -65,13 +70,28 @@ func serve(cctx *cli.Context) error {
 		return err
 	}
 
+	metricsMux := http.NewServeMux()
```

Comment:
Let's use a `DefaultServeMux` here and import `net/http/pprof` anonymously at the top of the file.